<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>971</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>3</horstretch>
    <verstretch>33</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>基金量化投研系统v0.11</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget_PGMS">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>20</horstretch>
        <verstretch>44</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_Combination">
       <attribute name="title">
        <string>产品组合管理</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>852</width>
             <height>958</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>收益</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>最大回撤</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Sharp比</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>最小范围</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>最大范围</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_returns_min">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximum">
                     <double>0.300000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_returns_max">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimum">
                     <double>0.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>10.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.100000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_maxdrawdown_min">
                    <property name="minimum">
                     <double>0.100000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.700000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_maxdrawdown_max">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimum">
                     <double>0.200000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>1.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_sharp_min">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximum">
                     <double>40.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.050000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_sharp_max">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="value">
                     <double>10.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>200</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_start_combination">
                  <property name="text">
                   <string>开始</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QCheckBox" name="checkBox_stock">
                  <property name="text">
                   <string>股票策略</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_compound">
                  <property name="text">
                   <string>复合策略</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_future_manage">
                  <property name="text">
                   <string>管理期货</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_event">
                  <property name="text">
                   <string>事件驱动</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_bond">
                  <property name="text">
                   <string>债券策略</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_macro">
                  <property name="text">
                   <string>宏观策略</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_combination_fund">
                  <property name="text">
                   <string>组合基金</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_relative_fund">
                  <property name="text">
                   <string>相对价值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_others">
                  <property name="text">
                   <string>其它策略</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWebEngineView" name="QWebEngineview_Combination_monte_markovitz" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 170, 127);</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QWebEngineView" name="QWebEngineview_Combination_Pie" native="true">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(170, 170, 127);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWebEngineView" name="QWebEngineview_Combination_Table" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(170, 170, 127);</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWebEngineView" name="QWebEngineview_Combination_Versus" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 170, 127);</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>QWebEngineview_Combination_monte_markovitz</zorder>
           <zorder></zorder>
           <zorder>QWebEngineview_Combination_Versus</zorder>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>黏贴</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>哈哈</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action_PyQt">
   <property name="text">
    <string>关于PyQt</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>其他</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
